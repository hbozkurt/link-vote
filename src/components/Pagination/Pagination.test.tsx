import * as React from "react";
import { create } from "react-test-renderer";
import { Pagination } from "./index";

describe("Pagination component", () => {
  test("Matches the snapshot", () => {
    const pagination = create(
      <Pagination
        currentPage={4}
        onChange={() => {}}
        totalItemsCount={96}
        itemsCountPerPage={5}
      />
    );
    expect(pagination.toJSON()).toMatchSnapshot();
  });
});
