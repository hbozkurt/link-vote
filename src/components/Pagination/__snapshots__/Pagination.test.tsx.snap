// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination component Matches the snapshot 1`] = `
<div
  className="flex justify-center text-gray-800"
>
  <span
    className="border w-8 h-8 p-4 flex justify-center items-center cursor-pointer rounded-l"
    onClick={[Function]}
  >
    <span>
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 256 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M31.7 239l136-136c9.4-9.4 24.6-9.4 33.9 0l22.6 22.6c9.4 9.4 9.4 24.6 0 33.9L127.9 256l96.4 96.4c9.4 9.4 9.4 24.6 0 33.9L201.7 409c-9.4 9.4-24.6 9.4-33.9 0l-136-136c-9.5-9.4-9.5-24.6-.1-34z"
        />
      </svg>
    </span>
  </span>
  <span
    className="border w-8 h-8 p-4 flex justify-center items-center cursor-pointer"
    onClick={[Function]}
  >
    <span>
      2
    </span>
  </span>
  <span
    className="border w-8 h-8 p-4 flex justify-center items-center cursor-pointer"
    onClick={[Function]}
  >
    <span>
      3
    </span>
  </span>
  <span
    className="border w-8 h-8 p-4 flex justify-center items-center bg-gray-900 text-white cursor-default"
    onClick={[Function]}
  >
    <span>
      4
    </span>
  </span>
  <span
    className="border w-8 h-8 p-4 flex justify-center items-center cursor-pointer"
    onClick={[Function]}
  >
    <span>
      5
    </span>
  </span>
  <span
    className="border w-8 h-8 p-4 flex justify-center items-center cursor-pointer"
    onClick={[Function]}
  >
    <span>
      6
    </span>
  </span>
  <span
    className="border w-8 h-8 p-4 flex justify-center items-center cursor-pointer rounded-r"
    onClick={[Function]}
  >
    <span>
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 256 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M224.3 273l-136 136c-9.4 9.4-24.6 9.4-33.9 0l-22.6-22.6c-9.4-9.4-9.4-24.6 0-33.9l96.4-96.4-96.4-96.4c-9.4-9.4-9.4-24.6 0-33.9L54.3 103c9.4-9.4 24.6-9.4 33.9 0l136 136c9.5 9.4 9.5 24.6.1 34z"
        />
      </svg>
    </span>
  </span>
</div>
`;
